core/unittest/reader/LogFileReaderUnittest.cpp

```diff
@@ -263,6 +341,30 @@ void LogFileReaderUnittest::TestLastMatchedLine() {
         APSARA_TEST_EQUAL_FATAL(std::string(testLog.data()), expectMatch);
         APSARA_TEST_EQUAL_FATAL(0, rollbackLineFeedCount);
     }
+    { // case single line, buffer size not big enough
```

Comment:
not big enough (no new line at the end of line)

Reply:
done